client/analytics/ad-tracking.js

```diff
@@ -62,15 +59,36 @@ const FACEBOOK_TRACKING_SCRIPT_URL = 'https://connect.facebook.net/en_US/fbds.js
 /**
  * Globals
  */
-if ( ! window._fbq ) {
-	window._fbq = []; // Facebook global
+if ( ! window.fbq ) {
+	setUpFacebookGlobal();
 }
 
 if ( ! window.uetq ) {
 	window.uetq = []; // Bing global
 }
 
+function setUpFacebookGlobal() {
+	const facebookEvents = window.fbq = function() {
+		if ( facebookEvents.callMethod ) {
+			facebookEvents.callMethod.apply( facebookEvents, arguments );
+		} else {
+			facebookEvents.queue.push( arguments );
+		}
+	};
+
+	if ( ! window._fbq ) {
+		window._fbq = facebookEvents;
+	}
+
+	facebookEvents.push = facebookEvents;
+	facebookEvents.loaded = true;
+	facebookEvents.version = '2.0';
+	facebookEvents.queue = [];
+}
+
 function loadTrackingScripts( callback ) {
+	hasFetchedScripts = true;
```

Comment:
I added this change to tighten the base case when this is used recursively to prevent fetching the scripts multiple times. This should prevent a bug that we've noticed very sporadically ([slightly more context](https://github.com/Automattic/wp-calypso/pull/2398#issuecomment-171469060)). I'll update the name to `hasStartedFetchingScripts`.